flow/packet-headers/icmp.yaml

```diff
@@ -39,9 +33,14 @@ components:
             description: >-
               code of ICMP with next fields
             format: integer
-            length: 8
+            length: 1
```

Comment:
The length of code is 8 bits. The description should include a comment that a code of 0 is used by both echo request and echo reply.